contrib/builder/rpm/armhf/centos-7/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/armhf/generate.sh"!
+#
+
+FROM multiarch/centos:7.2.1511-armhfp-clean
+
```

Comment:
can you add ``yum install -y yum-plugin-ovl`` in the generate script for here? This fixes an issue when building on overlay.